src/main/java/de/imi/mopat/model/Clinic.java

```diff
@@ -54,6 +54,8 @@ public class Clinic implements Serializable {
 
     @OneToMany(mappedBy = "clinic", cascade = CascadeType.ALL)
     private List<ClinicConfigurationMapping> clinicConfigurationMappings;
+    @OneToMany(mappedBy = "clinic")
```

Comment:
We do not need to add this to the model here. Instead we can add a method to the EncounterDao to fetch by clinic id.